newsfragments/2469.bugfix

```diff
@@ -0,0 +1 @@
+``dials.import``: ensure ``invert_rotation_axis=True`` is only applied once to each goniometer model
```

Comment:
Scope wise, this touches detector, scan and beam as well, though I note that goniometer is the only object where the reassignment is not idempotent since `invert_rotation_axis` is a toggle.

Reply:
Updated to note the more general scope of the changeset in the newsfragment.